modules/social_features/social_group/social_group.install

```diff
@@ -613,3 +613,30 @@ function social_group_update_8905() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Make group creators as a group managers by default.
+ */
+function social_group_update_8906() {
+  $config_factory = \Drupal::configFactory();
+
+  $group_types = [
```

Comment:
don't forget we have more group types ^^

Reply:
@ronaldtebrake I added these groups because these ones were used in the function I removed. Should I add "public" and "secret" group types as well?

Reply:
As long as you make sure it works for all group types i'm happy, can imagine it might not be needed but want to make sure you've tested this :D (i just did a quick glance at the code and it didnt become clear why haha)

Reply:
@ronaldtebrake Seems like you were right - these two group types ("open_group" && "secret_group") also should be updated as I didn't find any related updates for them. Fixed in the last commit.